tests/scripts/thread-cert/v1_2_test_csl_transmission.py

```diff
@@ -36,7 +36,9 @@
 
 CSL_PERIOD = 500 * 6.25  # 500ms
 CSL_TIMEOUT = 30  # 30s
-CSL_CHANNEL = 12
+CSL_CHANNEL = 13
+
+SECOND_CHANNEL = 12
```

Comment:
```suggestion
CSL_CHANNEL_1 = 13  # different from the default pan channel 11
CSL_CHANNEL_2 = 12
```

Reply:
is it necessary to test two different CSL channels?

Reply:
As above mentioned.